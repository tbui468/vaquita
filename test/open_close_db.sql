create database sol;
open sol;
close sol;
drop database sol;
exit;
