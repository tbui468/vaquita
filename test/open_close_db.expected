created database sol
opened database sol
closed database sol
dropped database sol
